drivers/modem/ublox-sara-r4.c

```diff
@@ -190,6 +212,86 @@ static int modem_atoi(const char *s, const int err_value,
 	return ret;
 }
 
+#if defined(CONFIG_MODEM_UBLOX_SARA_AUTODETECT_APN)
+
+const char *modem_sim_profiles =
```

Comment:
perhaps static here?

Reply:
I need to be able to change this on application startup. The least intrusive way I see is to use global scope - but if we should do it in another way, please comment.